oak/server/rust/oak_runtime/Cargo.toml

```diff
@@ -40,7 +40,7 @@ roughenough = "*"
 rustls = "*"
 serde = "*"
 serde_json = "*"
-sha2 = "*"
+sha2 = "0.8"
```

Comment:
Why is this necessary?

Reply:
When I regenerated `Cargo.lock`, it selected version 0.9.0, which seems to have introduced some breaking changes.